resolvers/webpack/index.js

```diff
@@ -265,7 +266,7 @@ function findExternal(source, externals, context) {
 
   // else, vanilla object
   for (var key in externals) {
-    if (!externals.hasOwnProperty(key)) continue
+    if (!has(externals, key)) continue
```

Comment:
(restarted as an explicit line review, mostly out of curiosity + to gain familiarity with the new feature)

So, I'm not anti-dependencies in general, but I don't understand why 

- someone would have file extensions of `toString` or `hasOwnProperty`
- someone would be surprised if things are broken when they monkeypatch JS core functions

Am I correct that these are the cases where `has` matters over just a plain ol' `hasOwnProperty` call or `in` operator? Because if that's true, I'd still like to revert to `hasOwnProperty` or really, just `extension in modifiers` because these are well-understood JS core idioms, and in this specific case, not practically distinguishable.